src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -249,6 +249,12 @@ def _path(ws, path):
             [PermissionLevel.CAN_USE, PermissionLevel.CAN_MANAGE],
             _simple,
         ),
+        (
+            "lakeview_dashboard",
```

Comment:
Could someone confirm that this is correctly implemented? I am missing links to docs on what resources should be defined

Reply:
Done, see integration test and below picture. The permissions were correct, but added a comment with more context

![Screenshot 2024-07-11 at 11 02 31](https://github.com/databrickslabs/ucx/assets/5946784/dabe8977-1390-44b0-a729-3fee12a91a6a)